src/hotspot/share/oops/markWord.hpp

```diff
@@ -41,8 +41,8 @@
 //
 //  64 bits:
 //  --------
-//  unused:25 hash:31 -->| unused_gap:1   age:4    biased_lock:1 lock:2 (normal object)
-//  JavaThread*:54 epoch:2 unused_gap:1   age:4    biased_lock:1 lock:2 (biased object)
+//  unused:32 hash:25 -->|  age:4    biased_lock:1 lock:2 (normal object)
+//  JavaThread*:54 epoch:2  age:4    biased_lock:1 lock:2 (biased object)
```

Comment:
This does not compute. Where did the `unused_gap` go in the second case? Is `epoch` now larger? Or does `JavaThread*` now takes 55 bits?

Reply:
Epoch and JavaThread in the biased case are already overlaid with hashcode. Hashcode and biased state doesn't happen at the same time. 32+25+4+1+2 == 64.

Reply:
Right, but I am asking about the _second case_, though, where: 54+2+4+1+2 == 63. Where does the `unused_gap` bit go?